Makefile

```diff
@@ -86,8 +88,12 @@ help: ## Display this help.
 ##@ Development
 
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+manifests: controller-gen crd-to-markdown ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd$(CRDDESC_OVERRIDE) webhook paths="./..." output:crd:artifacts:config=config/crd/bases && \
+	rm -f api/bases/* && cp -a config/crd/bases api/
```

Comment:
what's the purpose of this line? It just copies the CRD's into api/bases? Why is that needed?

Reply:
I just copied it from https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/blob/main/Makefile#L94-L97

all the changes of the Makefile came from there

Reply:
I've removed it